### PR TITLE
[stable/kube-lego] Adds 'update' permission for services

### LINKED ISSUE #2458: [stable/kube-lego] Error `put services kube-lego-nginx`
**Version of Helm and Kubernetes**:
kubernetes version: 1.7.5
helm version: 2.6.1

**Which chart**:
stable/kube-lego

**What happened**:
Error from pod:
```
time="2017-10-11T08:46:14Z" level=error msg="User \"system:serviceaccount:lego:encrypt-kube-lego\" cannot update services in the namespace \"lego\". (put services kube-lego-nginx)" context=provider provider=nginx
```

**What you expected to happen**:
have permissions to edit it's own service. I expect that the `put` verb is missing from the clusterrole definition.

**How to reproduce it** (as minimally and precisely as possible):
```
helm install stable/kube-lego --name encrypt -f values.yaml --namespace lego
```
Where contents of the `values.yaml` are:
```
config:
  LEGO_EMAIL: mail@ourcompany.com
  LEGO_URL: https://acme-v01.api.letsencrypt.org/directory
image:
  tag: canary
rbac:
  create: true
```

Then apply any ingress definition that is supposed to work with **lego**:

```
---
apiVersion: extensions/v1beta1
kind: Ingress
metadata:
  labels:
    app: chartmuseum
  name: chartmuseum
  namespace: 'internal-namespace-name'
  annotations:
    kubernetes.io/tls-acme: "true"
    kubernetes.io/ingress.class: "nginx"
spec:
  tls:
  - hosts:
    - chartmuseum.company.com
    secretName: chartmuseum-ingress-secret
  rules:
  - host: chartmuseum.company.com
    http:
      paths:
      - backend:
          serviceName: chartmuseum
          servicePort: 8080
        path: /
```

### PR DESCRIPTION
**Pull request by trobert2**
fixes #2458

### CODE REVIEW

**Comment by mgoodness** [comment]
Please edit the title to indicate which chart is being updated.

File: stable/kube-lego/values.yaml [hunk 1]
```diff
@@ -4,7 +4,7 @@
 config:
   ## Email address to use for registration with Let's Encrypt
   ##
-  # LEGO_EMAIL: my@email.tld
+  LEGO_EMAIL: my@email.tld
```
Comment (mgoodness) [inline stable/kube-lego/values.yaml]: This should remain commented, as people weren't paying attention to the fact that the email address had to be changed. Keeping it unset by default means deployment fails unless the user provides a value.

Ref: https://github.com/kubernetes/charts/pull/912

**Review by mgoodness — CHANGES_REQUESTED** [review]

File: stable/kube-lego/values.yaml (hunk 1 above)
Comment (trobert2) [reply stable/kube-lego/values.yaml]: I just edited this because otherwise I get a linting error. Will it pass the CI tests if I revert it?

Comment (mgoodness) [reply stable/kube-lego/values.yaml]: Unfortunately the tests won't pass, but we know about & expect the failure.

**Review by mgoodness — APPROVED** [review]

**Comment by mgoodness** [comment]
Needs a rebase & new version bump, then we'll get it merged.

Also, please allow maintainers to push to your branches in the future.

**Pull request merged**